src/Cli/func/Helpers/ZipHelper.cs

```diff
@@ -40,14 +40,15 @@ public static async Task<Stream> GetAppZipFile(string functionAppRoot, bool buil
             else if (GlobalCoreToolsSettings.CurrentWorkerRuntime == WorkerRuntime.Dotnet && buildOption == BuildOption.Remote)
             {
                 // Remote build for dotnet does not require bin and obj folders. They will be generated during the oryx build
-                return await CreateZip(FileSystemHelpers.GetLocalFiles(functionAppRoot, ignoreParser, false, new string[] { "bin", "obj" }), functionAppRoot, Enumerable.Empty<string>());
+                return await CreateZip(FileSystemHelpers.GetLocalFiles(functionAppRoot, ignoreParser, false, new string[] { "bin", "obj" }), functionAppRoot, Array.Empty<string>());
             }
             else
             {
-                var customHandler = await HostHelpers.GetCustomHandlerExecutable();
+                var customHandler = await HostHelpers.GetCustomHandlerExecutable(functionAppRoot);
```

Comment:
May be nothing actionable here right now. Would be good for a future workitem. We should try to localize logic for each command to their sub action class.